swift/Sources/FlatBuffers/string+extension.swift

```diff
@@ -0,0 +1,50 @@
+/*
+ * Copyright 2021 Google Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import Foundation
+// TODO: - Add docs
+extension String: Verifiable {
+
+  ///
+  /// - Parameters:
+  ///   - verifier:
+  ///   - position:
+  ///   - type:
+  /// - Throws:
+  public static func verify<T>(
+    _ verifier: inout Verifier,
+    at position: Int,
+    of type: T.Type) throws where T: Verifiable
+  {
+    
+    let range = try String.verifyRange(&verifier, at: position, of: UInt8.self)
+    let stringLen = range.start + range.count
+
+    #warning("check if this logic is fine")
+    if stringLen >= verifier.capacity {
+      throw FlatbuffersErrors.outOfBounds(position: position, end: verifier.capacity)
+    }
+
```

Comment:
The string is always going to be padded with at least one padding each time its added to the buffer. is this assumption correct? because again the fuzzer seems to crash there